lib/html/template/rose-bush/index.html

```diff
@@ -12,6 +12,9 @@
 <body>
 <div class="container-fluid">
 <div class="hero-unit">
+{% if logo -%}
+<img {{logo}} />
```

Comment:
See example in `etc/rose.conf.example` above. The variable `logo` should contain all the attributes required by the `img` tag.